packages/cypher-builder/src/expressions/list/ListExpr.ts

```diff
@@ -40,7 +49,15 @@ export class ListExpr implements CypherCompilable {
         return `[ ${serializedContent} ]`;
     }
 
+    /**
+     * @hidden
+     */
     public getCypher(env: CypherEnvironment): string {
         return this.serializeList(env, this.value);
     }
+
+    /* Access individual elements in the list via the ListIndex class*/
```

Comment:
Remember to use ** at the beginning to make it typedocs compatible

```suggestion
    /** Access individual elements in the list via the ListIndex class*/
```